scripts/deploy.sh

```diff
@@ -14,7 +15,7 @@ OSAD_DIR='/opt/rpc-extras/os-ansible-deployment'
 RPCD_DIR='/opt/rpc-extras/rpcd'
 
 # begin the bootstrap process
-cd "${OSAD_DIR}"
+cd ${OSAD_DIR}
```

Comment:
ya, looking at it, the quotes were only removed for the statically set local variables, so we are fine here